modules/core/src/copy.cpp

```diff
@@ -94,14 +94,30 @@ copyMask_<uchar>(const uchar* _src, size_t sstep, const uchar* mask, size_t mste
         {
             v_uint8 v_zero = vx_setzero_u8();
 
-            for( ; x <= size.width - v_uint8::nlanes; x += v_uint8::nlanes )
+            for( ; CV_SIMD_IS_LT_ALIGNED_UNROLLED(x, size.width, v_uint8, 2); x += v_uint8::nlanes * 2)
```

Comment:
I think it could be a really long story to find out a best way to unify SIMD loops. So since it's completely unrelated to copyMask improvement I prefer to retain this discussion out of the PR scope.
@alalek WDYT?